config/production.json

```diff
@@ -1,13 +1,12 @@
 {
-  "host": "youpin-api-app.feathersjs.com",
-  "port": 80,
-  "nedb": "NEDB_BASE_PATH",
+  "host": "localhost",
+  "port": 9000,
+  "mongodb": "mongodb://mongodb:27017/youpin",
   "public": "../public/",
   "auth": {
     "token": {
-      "secret": "FEATHERS_AUTH_SECRET"
+      "secret": "f+7jgJJG7q6Uyx1rrXG+nWqG6f3jn7UcY5h1DYVY4Te5WH9J+16mR58GfdacxpQQy9l96ygCj5to23Fz7bcktQ=="
```

Comment:
We should not commit token secret into repo. Actually, we should create and commit `production.template.json`. When we run it, we copy it to `production.json` (which is not included in git repo) and change content inside.